README.md

```diff
@@ -72,6 +72,27 @@ For example:
 # Hello {props.name}
 ```
 
+#### JSDoc in MDX with Prettier
+
+If you’re using Prettier, you’ll need to ignore any JSDoc comments until it is
+supported (issues [MDX 3](https://github.com/prettier/prettier/issues/16457)
+and
+[MDX v2+ JSDoc comments](https://github.com/prettier/prettier/issues/16457)):
```

Comment:
I think it would be better to only link this issue: https://github.com/prettier/prettier/issues/12209

Reply:
done in:

- f3a07bd0256f08a4c858afe327c4b5dc5c3e7167
- e6aae5d19b7ef5dad3f599a83648af6c922b0420